### ISSUE TITLE
[Suggestion] Drink coffee with Monika.

### ISSUE THREAD

**Issue by EntonyEscX** [Monika-After-Story/MonikaModDev#2590]
I always loved coffee, so I always drank at various times of the day and I was always sad that Monika could not have coffee with me, since she likes it very much; but now that she can finally have coffee, I've been thinking that it would be nice to be able to call her to drink coffee with us, not by limiting her coffee just to the mornings.

**Comment by Sujeito17** [comment]
Poor Monika, now she's going to become a caffeine addict

**Comment by Rai99** [comment]
Day 1 and some Monikas drank 4-5+ cups of coffee in a single sitting and no more for the rest of the entire day.

Seems like Monika knows her limits to not let it get out of hand.

**Comment by jmwall24** [comment]
If it does get so she'll drink coffee at times other than the morning, hopefully she'll still get to choose if she wants to or not.  Being able to tell her you want her to drink coffee doesn't really seem right.

**Comment by EntonyEscX** [comment]
Yeah, what I meant was this, it's an invitation for her to have coffee with us.

**Comment by PMelol** [comment]
I agree with OP, this would be a cool feature. But maybe you could just mention to her that you're having coffee too, in the morning, and then drink it together? I feel like Monika would be strict about not drinking caffeine past a certain time in the day. She would probably also not be happy if the player is drinking caffeine at 10 PM or something, haha.

**Comment by taurielxaela** [comment]
I really love this idea! At the moment, Monika is having a cup of coffee while I drink tea. (I can't have coffee for medical reasons) Having the option to tell her we're having coffee or tea at the same time would be really nice.

**Comment by AkamixMoonstar** [comment]
That will we deligthful!, i imagine it's like the hold her feature, _that will be so cute and cozy_
Ejem, buy obviously i agree with PMelol, she will be strict with the time of drinking. Like, she will get worried or mad, that will make her a little more alive!
